x-pack/test/api_integration/apis/synthetics/get_monitor.ts

```diff
@@ -242,7 +242,6 @@ export default function ({ getService }: FtrProviderContext) {
             revision: 1,
             locations: [LOCAL_LOCATION],
             name: 'Test HTTP Monitor 044',
-            labels: {},
```

Comment:
We recently added labels but are they gone now?

Reply:
it's because we filter out empty fields